src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,12 +4,15 @@
  * No ending dot
  */
 public class OtherClass {
```

Comment:
**Reporter**: Findbugs
**Rule**: NM_FIELD_NAMING_CONVENTION
**Severity**: INFO
**File**: src/main/java/se/bjurr/violations/lib/example/OtherClass.java L6
**Source**: se.bjurr.violations.lib.example.OtherClass

Field names should start with a lower case letter


  <p>
Names of fields that are not final should be in mixed case with a lowercase first letter and the first letters of subsequent words capitalized.
</p>


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<783602010>*